packages/react-native-sdk/src/components/ReactionsModal.tsx

```diff
@@ -0,0 +1,87 @@
+import { StreamReaction } from '@stream-io/video-client';
+import { Pressable, StyleSheet, View, Text } from 'react-native';
+
+import { theme } from '../theme';
+import { useCallback } from 'react';
+import { useCall } from '@stream-io/video-react-bindings';
+import { defaultEmojiReactions } from '../constants';
+
+const reactions: StreamReaction[] = [
+  {
+    type: 'reaction',
+    emoji_code: ':like:',
+    custom: {},
+  },
+  {
+    // TODO OL: use `prompt` type?
```

Comment:
What is this, let's remove this

Reply:
This was a comment in the React SDK by Oliver, forgot to remove it, will remove it now.